src/lib/middleware/secure-headers.ts

```diff
@@ -13,12 +13,18 @@ const secureHeaders: (config: IUnleashConfig) => RequestHandler = (config) => {
             },
             contentSecurityPolicy: {
                 directives: {
-                    defaultSrc: ["'self'", 'cdn.getunleash.io', 'gravatar.com'],
+                    defaultSrc: [
+                        "'self'",
+                        'cdn.getunleash.io',
+                        'gravatar.com',
+                        ...config.additionalCspAllowedDomains?.defaultSrc,
```

Comment:
Would this crash if `additionalCspAllowedDomains` is undefined?

Reply:
No, because if additionalCspAllowedDomains is undefined `parseCspEnvironmentVariables()` will trigger and provide defaults.

Reply:
Oh right! Maybe `additionalCspAllowedDomains` can be required in `IUnleashConfig` then, to remove the `?`s.